jobs/cloud_controller_ng/spec

```diff
@@ -85,8 +85,8 @@ provides:
   - ccdb.read_timeout
   - ccdb.ssl_verify_hostname
   - ccdb.ca_cert
-- name: directories_to_backup
-  type: directories_to_backup
+- name: blobstore_directories
```

Comment:
Since this link is still only actually used by the backup and restore workflow, I would prefer if that was still captured in its name. Maybe `blobstore_directories_to_backup` would be the best of both worlds here?

The thing is there are other blobstore directories like the app resource cache and this current naming kind of implies this is the comprehensive list. We've found that other release authors find and consume links like this for purposes that we didn't originally intend so want to make sure the name is clear.

Reply:
Fair point. Thank you for sharing your experience of other release authors finding and consuming links for different purposes.